lib/modules/manager/flux/extract.ts

```diff
@@ -197,6 +207,29 @@ function resolveResourceManifest(
         deps.push(dep);
         break;
       }
+      case 'OCIRepository': {
+        const container = resource.spec.url?.replace('oci://', '');
+        const dep: PackageDependency<FluxManagerData> = {
+          depName: container,
+        };
+        if (resource.spec.ref?.digest) {
+          dep.currentDigest = resource.spec.ref.digest;
+          dep.currentValue = 'latest';
+          dep.datasource = DockerDatasource.id;
+        } else if (resource.spec.ref?.tag) {
+          const containerTag = getDep(container + ':' + resource.spec.ref?.tag);
```

Comment:
```suggestion
          dep = getDep(container + ':' + resource.spec.ref?.tag);
```
why not using the returned value directly. also pass the no auto-replace template flag.

i think it's also a good idea to pass registry aliases.

Reply:
@viceice 

1. `dep` is a `const` that is used in all the conditional blocks, what you suggest won't work. What do you mean by that?
2. I tried pass the registry aliases but I received the error:

```
  ● modules/manager/flux/extract › extractPackageFile() › extracts OCIRepository with a tag

    TypeError: Cannot read properties of undefined (reading 'registryAliases')

      221 |           }
      222 |         } else if (resource.spec.ref?.tag) {
    > 223 |           const containerTag = getDep(`${container}:${resource.spec.ref.tag}`, false, config.registryAliases);
          |                                                                                              ^
      224 |           dep.autoReplaceStringTemplate =
      225 |             '{{#if newValue}}:{{newValue}}{{/if}}{{#if newDigest}}@{{newDigest}}{{/if}}';
      226 |           dep.currentDigest = containerTag.currentDigest;

      at resolveResourceManifest (lib/modules/manager/flux/extract.ts:223:94)
      at extractPackageFile (lib/modules/manager/flux/extract.ts:264:14)
      at Object.<anonymous> (lib/modules/manager/flux/extract.spec.ts:519:40)
```

Reply:
ok, remove `registryAliases` for now, can be done later and would need a lot more changes. you can checkout the dockerfile manager, or just search for `registryAliases` in codebase

Reply:
I did search around a bit on that while I was trying to make it work but ran into those issues. Maybe a small refactor is needed for it to fall into place.